lib/process.cc

```diff
@@ -114,11 +116,13 @@ namespace crucible {
 	template<>
 	struct ResourceHandle<Process::id, Process>;
 
+#ifndef SYS_GETTID
 	pid_t
 	gettid()
 	{
 		return syscall(SYS_gettid);
```

Comment:
I think I prefer just making all the gettid() calls in bees use the `crucible::` namespace explicitly.

Reply:
This was also my first thought, so I'd vote for this.

Reply:
@Kriskras99 Could you refine your patch so we can get this fixed? @Zygo is a busy man and I'd prefer to have this in the official repository over managing custom patches in a distro build... ;-)